site/en/getstarted/cluster/install_cluster-milvusoperator.md

```diff
@@ -7,27 +7,27 @@ group: install_cluster-docker.md
 summary: Learn how to install Milvus cluster on Kubernetes using Milvus Operator
 ---
 
-# Install Milvus Cluster
-
-{{fragments/installation_guide_cluster.md}}
-
 {{tab}}
 
-## Create a Kubernetes Cluster
+# Install Milvus Cluster with Milvus Operator
 
-If you have already deployed a K8s cluster for production, you can skip this step and proceed directly to [deploy Milvus Operator](install_cluster-milvusoperator.md#Deploy-Milvus-Operator). If not, you can follow the following steps to quickly create a K8s for testing, and then use it to install a Milvus cluster with Milvus Operator. This tutorial introduces two ways to create a Kubernetes cluster:
+Milvus Operator is a solution that helps you deploy and manage a full Milvus service stack to target K8s clusters. The stack includes all Milvus components and relevant dependencies like etcd, Pulsar and MinIO. This topic introduces how to deploy a Milvus cluster with Milvus Operator on Kubernetes.
```

Comment:
Milvus Operator is a solution that helps you deploy and manage a full Milvus service stack to target **Kubernetes (K8s)** clusters. The stack includes all Milvus components and relevant dependencies like etcd, Pulsar and MinIO. This topic introduces how to deploy a Milvus cluster with Milvus Operator on **K8s**. 

You can use the abbreviation of K8s throughout in the rest of the article.